### PR TITLE
translation string in vendor dashboard and changing more product into…

### PR DESCRIPTION
**Pull request by shabnam611**
… more products

### CODE REVIEW

File: includes/template-tags.php [hunk 1]
```diff
@@ -330,7 +330,7 @@ function dokan_get_dashboard_nav() {
     );
 
     $settings = array(
-        'title' => __( 'Settings <i class="fa fa-angle-right pull-right"></i>', 'dokan-lite'),
+        'title' => sprintf(__("%s <i class='fa fa-angle-right pull-right'></i>"), __( 'Settings', 'dokan-lite')),
```
Comment (sabbir1991) [inline includes/template-tags.php]: Don't need to add `__()` function inside `sprintf` function.

Comment (shabnam611) [reply includes/template-tags.php]: Thanks for the feedback, I corrected it. Please check.

Comment (ediamin) [reply includes/template-tags.php]: Use single quote outside and double quote for attributes. like 
`'%s <i class="fa fa-angle-right pull-right"></i>'`

Comment (sabbir1991) [reply includes/template-tags.php]: Please also remove extra `()` inside `sprintf`

**Pull request merged**